docs/source/datasets.rst

```diff
@@ -92,6 +92,14 @@ click the link provided in the "Module" column.
 | salami_           | Salami              | - audio: ❌         | - :ref:`sections`         | 1359   |
 |                   |                     | - annotations: ✅   |                           |        |
 +-------------------+---------------------+---------------------+---------------------------+--------+
+| saraga_           | Saraga              | - audio: ✅         | - :ref:`tempo`            | 305    |
+|                   |                     | - annotations: ✅   | - :ref:`phrases`          |        |
+|                   |                     |                     | - :ref:`f0`               |        |
+|                   |                     |                     | - :ref:`f0` (vocal)       |        |
+|                   |                     |                     | - :ref:`sections`(samas)  |        |
```

Comment:
small formatting error - needs a space between `sections` and (samas)

Reply:
Thanks! Fixed now with the rest of the documentation fixes.